### PR TITLE
Shashank/new benchmarks

### PR DESCRIPTION
**Pull request by ShashankMosaicML**
Adding new benchmarks for large scale pretraining (>=128 GPUs), as well as adding minor updates to the benchmarking scripts.

### CODE REVIEW

File: scripts/train/benchmarking/submit_benchmarks.py [hunk 1]
```diff
@@ -420,11 +420,11 @@ def run_config(config: Tuple[str, int, int, str, str, int, str],
     if args.data_remote is None:
         command += f"""
             cd llm-foundry/scripts
-            python data_prep/convert_dataset_hf.py --dataset c4 --data_subset en --out_root ./my-copy-c4 --splits train_small val_small --concat_tokens {max_seq_len} --tokenizer gpt2 --eos_text '<|endoftext|>'
```
Comment (vchiley) [inline scripts/train/benchmarking/submit_benchmarks.py:423]: why not allow user to set max seq len?

Comment (ShashankMosaicML) [reply scripts/train/benchmarking/submit_benchmarks.py:423]: Good catch, let me revert it.

Comment (ShashankMosaicML) [reply scripts/train/benchmarking/submit_benchmarks.py:423]: Done.

**Review by vchiley — APPROVED** [review]
👍

**Pull request merged**